src/mocks/db.json

```diff
@@ -3570,14 +3570,6 @@
       "postId": "2",
       "acceptedTermsAndConditions": true,
```

Comment:
Actually we don't need to store this value: `acceptedTermsAndConditions`. It is only for validation